backend/proxy/shared-location.conf

```diff
@@ -6,9 +6,8 @@
     proxy_cache webapp_cache;
     proxy_cache_use_stale error timeout http_500 http_502 http_503 http_504;
     proxy_cache_key $uri$is_args$args$http_language;
-    if ($request_method = 'PURGE') {
```

Comment:
Is there ever a case where a PURGE request won't come from the allow list? I'm curious on the original intent of allowing purges from any domain

Reply:
No, noting the TODO comment, we just never got back to it until now.